streaming/src/test/scala/org/apache/spark/streaming/InputStreamsSuite.scala

```diff
@@ -278,11 +294,14 @@ class InputStreamsSuite extends TestSuiteBase with BeforeAndAfter {
     }
   }
 
-  def testFileStream(newFilesOnly: Boolean) {
+  def testFileStream(newFilesOnly: Boolean, depth :Int = 1) {
```

Comment:
This should be `depth: Int = 1`. There is at least another instance above where `name : Type` is written; might as well standardize to `name: Type` everywhere.